src/common/winterStorageAreaCard/WinterStorageAreaCard.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+import Card from '../card/Card';
+import styles from '../harborCard/harborCard.module.scss';
```

Comment:
Shouldn't this be `./winterStorageAreaCard.module.scss`?

Reply:
Indeed it should. Fixed.